Assets/MixedRealityToolkit/Input/Scripts/Cursor/AnimCursorDatum.cs

```diff
@@ -4,7 +4,7 @@
 using System;
 using UnityEngine;
 
-namespace HoloToolkit.Unity.InputModule
+namespace MixedRealityToolkit.Input.Cursor
```

Comment:
`MixedRealityToolkit.Input.Cursor` conflicts with `UnityEngine.Input.Cursor` as well.

Reply:
Should we rename Cursor to MRCursor then?
Although, by renaming the Cursor class to BaseCursor, there were no other conflicts as we have no conflicting classes.